README.md

```diff
@@ -0,0 +1,215 @@
+
+```properties
+# Download the image
+docker pull petalslink/petals-deployer:latest
+
+# Start in detached mode (with default config)
+docker run -d -p 80:80 --name petals-deployer petalslink/petals-deployer:latest
+
+# Start example #1
+docker run \
+	-d -p 80:80 --name petals-deployer \
+	-v /home/user/Petals/artifacts/:/tmp/artifact/ \
+	-e "ENV_DEPLOY_FILE=/tmp/artifact/filesToDeploy.txt" \
+	-e "ENV_CLI_HOST=172.17.0.2" \
+	petalslink/petals-deployer:latest
+
+# Start example #2
+docker run \
+	-d -p 80:80 --name petals-deployer \
+	-e "ENV_CLI_HOST=172.17.0.5" \
+	-e "ENV_DEPLOY_URLS=mvn:org.ow2.petals/petals-bc-soap/LATEST/zip;mvn:org.ow2.petals/petals-bc-ftp/LATEST/zip" \
+	petalslink/petals-deployer:5.2.0
+
+# Verify the container was launched
+docker ps
+
+# Watch what happens/happened
+docker logs petals-deployer
+
+# Verify the ports used on the host (example)
+sudo lsof -i :80
+
+# Get information about the container
+docker inspect petals-deployer
+
+# Introspect the container
+docker exec -ti petals-deployer /bin/bash
+```
+
+The example shows how to get the last version.
+You can obviously change the version. Each Petals container release has a corresponding deployer release, with a matching version. As an example, to get a deployer for **Petals ESB 5.1.0** (which would actually be **Petals CLI 3.0.0**), just type in `docker pull petalslink/petals-deployer:5.1.0`.
+> You can choose a specific **Petals CLI** version by building your own image
+
+## Run parameters
+
+When running this deployer, you must specify it which artifacts to deploy on which Petals ESB container instance.
+> Please note that service assemblies require components which may require shared libraries. Be sure to deploy in the correct order. Every artifact will be started (if possible) after being deployed.     
+
+You must use the following parameters :
+
+| Argument | Optional | Default | Description |
+| -------- | :------: | :-----: | ----------- |
+| CLI_PETALS_HOST | yes | `localhost` | The petals container hostname. |
+| CLI_PETALS_PORT | yes | `7700` | The petals container JMX port. |
+| CLI_PETALS_USER | yes | `petals` | The petals container user to use. |
+| CLI_PETALS_PASS | yes | `petals` | The petals container password to use.  |
+| DEPLOY_URLS | this and/or DEPLOY_FILE must be set | - | The artifact list (as String) to be deployed, separated by the semicolon character (`;`). This artifact list will be processed **before** DEPLOY_FILE. **A valid CLI *deploy* format is expected, see below.**  Example: `mvn:<url1>;mvn:<url2>;file:<file1>` |
+| DEPLOY_FILE | this and/or DEPLOY_URLS must be set | - | The artifact list (as file) to be deployed, each file line is treated separately. Remember that to be able to access a local file, **a volume has to be mounted**. This artifact list will be processed **after** DEPLOY_URLS. **A valid CLI *deploy* format is expected, see below.** Example: `/tmp/artifact/filesToDeploy.txt`|
```

Comment:
Invalid example. It should be file:/tmp....

Reply:
I'm using :
```sh
  while ... do
...
  done < "$DEPLOY_FILE"
```
to feed the file and actually `file:/tmp....` is not accepted but `/tmp/...` is.
How should I feed the file to accept `file:/` ?